receiver/postgresqlreceiver/metadata.yaml

```diff
@@ -1,20 +1,37 @@
 name: postgresqlreceiver
 
+resource_attributes:
+  postgresql.database:
+    description: The name of the database.
+    type: string
+  postgresql.table:
```

Comment:
These should probably have `.name` on the end, so that it's possible to reuse the namespaces (eg. `postgresql.table.id`)